docs/6.x/changelog.md

```diff
@@ -32,6 +32,13 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.0.4 (August 15th, 2019)
+
+#### Bugfixes
+
+* Fix an issue with ssh'ing into some nodes via Cluster Control Panel.
+* Fix an issue with some audit events not being returned when using custom date picker.
```

Comment:
not being returned -> filtered out?

Reply:
I think these two mean different things and "not being returned" fits better in this case.